examples/gno.land/p/demo/ownable/exts/authorizable/authorizable.gno

```diff
@@ -0,0 +1,90 @@
+// Package authorizable is an extension of p/demo/ownable;
+// It allows the user to instantiate an Authorizable struct, which extends
+// p/demo/ownable with a list of users that are authorized for something.
+// By using authorizable, you have a superuser (ownable), as well as another
+// authorization level, which can be used for adding moderators or similar to your realm.
+package authorizable
+
+import (
+	"std"
+
+	"gno.land/p/demo/avl"
+	"gno.land/p/demo/ownable"
+	"gno.land/p/demo/ufmt"
+)
+
+type Authorizable struct {
+	*ownable.Ownable           // owner in ownable is superuser
+	authorized       *avl.Tree // std.Addr > struct{}{}
+}
+
+func NewAuthorizable() *Authorizable {
```

Comment:
I don't think so; you will either define an Ownable, or an Authorizable. Ownable provides only one level of administration, while Authorizable will have two. In what case do you see `NewAuthorizableWithOwnable` being used?

I don't think you will try to compose your Authorizable with someone else's Ownable at any time, especially since Ownable always needs to be defined as an unexported variable, so no one can just change the admin address.

Reply:
One use case would be to define a realm that has an instance of Ownable. The owner of this realm may want to, at some point use this single ownable instance to serve as the basis for of an authorizable instance used by other realms from the same author. I was just suggesting it for convenience. Of course the user can still do this as Ownable is an exported field.